backend/wayland/backend.c

```diff
@@ -117,6 +118,38 @@ struct wlr_wl_backend_output *wlr_wl_output_for_surface(
 	return NULL;
 }
 
+void wlr_wl_output_layout_get_box(struct wlr_wl_backend *backend,
+		struct wlr_box *box) {
```

Comment:
What the heck is this?

Reply:
Yeah, I needed to re-implement this function. Basically we send absolute events, which means pointer events are mapped to the whole output layout. Each window acts like a tablet tool, so we need to re-map each "tablet tool" to the right region.

Reply:
Aren't pointer events (even absolute ones) relative to the surface the pointer is over?

Reply:
There's no "surface hovered by the pointer".

Imagine a two-output setup with one tablet pad. If you click on the left part of the tablet pad, it ends up on the left output - same for the one on the right.

The wayland backend, started with two windows, has two outputs and two tablet pads. But you want to map each tablet pad to an output, not each tablet pad to the whole output layout.